src/ArcGISEarth.AutoAPI.Examples/MainWindow.xaml

```diff
@@ -1,65 +1,419 @@
 ﻿<Window x:Class="ArcGISEarth.AutoAPI.Examples.MainWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:i="http://schemas.microsoft.com/xaml/behaviors"
         xmlns:local="clr-namespace:ArcGISEarth.AutoAPI.Examples"
-        Title="ArcGISEarth.AutoAPI.Examples" Height="620" Width="800" ResizeMode="CanMinimize">
-    <Window.DataContext>
-        <!--Use MainWindowViewModel as the window data context-->
-        <local:MainWindowViewModel/>
-    </Window.DataContext>
-    <Grid>
-        <Grid.RowDefinitions>
-            <RowDefinition Height="60"/>
-            <RowDefinition Height="380"/>
-            <RowDefinition Height="*"/>
-        </Grid.RowDefinitions>
-        <Grid Grid.Row="0" Margin="10,10,20,0">
-            <!-- Automation API url -->
-            <TextBlock Margin="20,0,0,0" Text="Automation API Url" VerticalAlignment="Center"/>
-            <TextBox Margin="140,0,0,0" Text="{Binding APIUrl}" Padding="2,2,0,0" Width="540" Height="24" HorizontalAlignment="Left"/>
-            <Button Margin="0,0,0,0" Command="{Binding ResetCommand}" Content="Reset" Width="60" Height="24" VerticalAlignment="Center" HorizontalAlignment="Right"/>
```

Comment:
There is no control to enter API url on new UI, so users cannot customize the URL of the API. We need to discuss this.